ui/lib/config-ui/addon/routes/login-settings/rule/details.js

```diff
@@ -4,5 +4,28 @@
  */
 
 import Route from '@ember/routing/route';
+import { service } from '@ember/service';
 
-export default class LoginSettingsRuleDetailsRoute extends Route {}
+export default class LoginSettingsRuleDetailsRoute extends Route {
+  @service store;
+
+  async model() {
+    try {
+      const { name } = this.paramsFor('login-settings.rule');
+      if (!name) return null;
```

Comment:
Returning null would then mean there's no data returned to this route. If there's no name param we should handle this in the `beforeModel` hook and transition them to the list view instead